README.md

```diff
@@ -1,2 +1,10 @@
 # InteractiveProgramming
 This is the base repo for the interactive programming project for Software Design, Spring 2016 at Olin College.
+
+Our Project Proposal:
+https://docs.google.com/document/d/10Eax7S3gga7mB4fEm4NYQ3jXghqH5RktQyJAh4wDo6k/edit?usp=sharing
```

Comment:
Good proposal, sounds like a reasonable plan heading forward, especially your consideration of limitations.